addons/hr/models/hr_employee.py

```diff
@@ -267,6 +271,14 @@ def _compute_work_permit_name(self):
             permit_no = '_' + employee.permit_no if employee.permit_no else ''
             employee.work_permit_name = "%swork_permit%s" % (name, permit_no)
 
+    @api.depends('activity_state', 'activity_ids')
+    def _compute_activity_plans_ids(self):
+        for employee in self:
+            if not employee.activity_state:
+                employee.activity_plans_ids = False
+                continue
+            employee.activity_plans_ids = employee.activity_ids.plan_id
```

Comment:
It means that if you have an open activity, you'll take the plan for done activities as well I think, which is not correct regarding the field name (Ongoing Activity Plans).

Reply:
Can't you simply check employee activities that are ongoing and linked to some plan ? Don't understand how we end up with such complex code.